### PR TITLE
ran 2to3 on entire project

### PR DESCRIPTION
**Pull request by michaelzhang608**
I had incorrectly created a branch off of `develop` instead of `master` earlier today, but this one is based off of `master`.

### CODE REVIEW

**Comment by isedwards** [comment]
We're getting a helpful message:

> #### A build service has not been set up
> We have detected a top-level Dockerfile. Pick from [apps that can perform automatic builds.](https://github.com/marketplace/category/container-ci)

There is also a [`.travis.yml`](https://github.com/IDEMSInternational/molo-iogt/blob/develop/.travis.yml) in the root of the repository.

I understand the process if setting up [Travis-CI](https://travis-ci.com/) to autmoatically run the tests each time a new pull request is received, but I'm not sure about using docker (as suggested in the message).

We can see that [unicef/molo-iogt](https://github.com/unicef/molo-iogt/) was using travis (e.g. see [here](https://github.com/unicef/molo-iogt/runs/977221405))

Any thoughts on whether we should also try to set up an Azure/Docker/Kubernetes build and whether we would then use this to run the tests?

**Comment by nathanbaleeta** [comment]
For testing, IoGT is deployed to a test Kubernetes cluster (multiple servers), Rancher is used to deploy docker containers (which consist of the IoGT code).

A test area dedicated to IDEMS can be setup for us and they would walk us through on how to deploy their updated code. It would be great if @Ian you join the Monday call with New York team. We would be able to figure this part out.

**Comment by isedwards** [comment]
Thank you @nathanbaleeta, I'd be very happy to join the call - It would be good to understand the deployment process and it sounds like we won't need to setup our own kubernetes environment for testing which is ideal. 

I've added an issue for setting up CI on this IDEMS fork of the project.

File: iogt/views.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import urllib
```
Comment (isedwards) [inline iogt/views.py:1]: @michaelzhang608 looking in views.py, the original `import urllib` (and also the suggested replacements `import urllib.request, urllib.parse, urllib.error`) are not used.

Could you add a second commit to delete this line with a commit message like "Removed unused import"

**Review by isedwards — CHANGES_REQUESTED** [review]
All of the suggestions from the `2to3` tool are welcome changes - but they also show that the original code is already valid Python 3.

This was also confirmed by @nathanbaleeta 's pull request to the original unicef/molo-iogt `develop` branch ([here](https://github.com/unicef/molo-iogt/pull/20))...

Although we don't yet have automated tests running at IDEMSInternational/molo-iogt, the tests did run at the unicef fork of the repository and we saw there that the automated tests are set up to [run on both Python 2.7 and 3.6](https://travis-ci.com/github/unicef/molo-iogt/builds/181488087).

I've suggest one minor change (deletion) to be made in an additional commit before merging this to our local `master`. Then we should test that everything runs locally with Python 3.7.

File: iogt/views.py (hunk 1 above)
Comment (michaelzhang608) [reply iogt/views.py:1]: @isedwards Ok, done.

**Comment by michaelzhang608** [comment]
@nathanbaleeta If possible, could I sit in to that call on Monday?

**Comment by isedwards** [comment]
@nathanbaleeta are you happy to merge this? We can then use it for https://github.com/IDEMSInternational/molo-iogt/issues/2#issuecomment-682020676 and https://github.com/IDEMSInternational/molo-iogt/issues/3#issuecomment-682023459.

@michaelzhang608 It looks like we should already have the meeting invite for Monday, but I will forward it to your email. See [this link](https://www.timeanddate.com/worldclock/meetingdetails.html?year=2020&month=8&day=31&hour=14&min=0&sec=0&p1=115&p2=250&p3=136&p4=179) to confirm the meeting time.

**Review by isedwards — APPROVED** [review]

**Comment by nathanbaleeta** [comment]
> @nathanbaleeta If possible, could I sit in to that call on Monday?

I'll ask Laurie to send you an invite as well.

**Pull request merged**